infrastructure/lambda/template.yaml

```diff
@@ -195,6 +195,225 @@ Mappings:
       localdev: HOURS
 
 Resources:
+  #################
+  # START OJ-2862 #
+  #################
+  FixSessionTTLStateMachine:
+    Type: AWS::Serverless::StateMachine
+    Properties:
+      Name: !Sub ${AWS::StackName}-FixSessionTTLStateMachine
+      Type: EXPRESS
+      DefinitionUri: ../../step-functions/fix-session-ttl.asl.json
+      DefinitionSubstitutions:
+        TimeFunction: !Ref TimeFunction
+        SessionTable: !Ref AddressTable
```

Comment:
Mind just doing a find and replace and updating this to AddressTable? Just confused myself about which table we're looking at here